extensions/amp-story-page-attachment/0.1/amp-story-open-page-attachment.js

```diff
@@ -94,14 +97,20 @@ const ctaLabelFromAttr = (element) =>
  * @param {?string} label
  * @return {?string}
  */
-const openLabelOrFallback = (element, attachmentEl, label) =>
-  attachmentEl.tagName === 'AMP-STORY-SHOPPING-ATTACHMENT'
-    ? localize(element, LocalizedStringId_Enum.AMP_STORY_SHOPPING_CTA_LABEL)
```

Comment:
Why not use the `localize` helper?

Reply:
The `localize` helper is used inside `amp-story` because part of it is "instantiate a new LocalizationService if not instantiated already". On extensions we don't want to add the service in the bundle, just fetch it, so we can't use that `localize` helper or we would be bundling it into all the extensions that need it.